client/extensions/woocommerce/app/products/products-list-table.js

```diff
@@ -39,12 +42,12 @@ const ProductsListTable = ( { translate, products, site, isRequesting } ) => {
 				className={ classNames( { 'is-requesting': isRequesting } ) }
 				horizontalScroll
 			>
-				{ products &&
+				{ !! products.length &&
```

Comment:
Why is the `!!` needed?

Reply:
Converts to boolean - otherwise the component prints a `0`

Reply:
Ah - of course - I was merely thinking of the conditional and not the resulting rendering - ty

Reply:
From a code readability standpoint, I like `Boolean( value )` better than `!! value`, but that's probably more of a personal preference.